specs/ledger/hs/src/Ledger/Update.hs

```diff
@@ -113,9 +115,6 @@ upSigData = lens
     & upSwVer .~ sv
   )
 
-inRange :: Ord a => a -> a -> a -> Bool
-inRange l u x = l <= x && x <= u
-
 -- | Test if a pair is present in a map.
 inMap :: (Ord k, Eq v) => k -> v -> Map k v -> Bool
 inMap k v m = case Map.lookup k m of
```

Comment:
When I do `stack build` I get the following error
```
Update.hs:455:9: error: [-Wincomplete-patterns, -Werror=incomplete-patterns]
        Pattern match(es) are non-exhaustive
        In a case alternative: Patterns not matched: []
        |
    455 |         case fads of
        |         ^^^^^^^^^^^^...
```

Reply:
In the meantime try running `stack` with `--flag '*:development'`.